### ISSUE TITLE
Show other mutation attributes as separate colours in 3D datashade?

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/tsqc#123]
Datashader supports shading multiple attributes of each pixel in different colours. I wonder if we could shade num_inheritors as another dimension? Might take some fiddling, but the examples here look like it might work:

https://datashader.org/getting_started/Pipeline.html#colormapping

https://datashader.org/getting_started/Pipeline.html#colormapping-3d-categorical-data

These are categorial though, so maybe num_inheritors wouldn't work. We could imagine having intron/exon or something as a categorial to shade by. Worth playing with a bit anyway I think.

**Comment by savitakartik** [comment]
Yes, this would be really cool with genome annotations! I'll have a go at this once we have the genome tracks in place.
